src/pages/Report_s1/index.tsx

```diff
@@ -0,0 +1,288 @@
+import { useState } from 'react'
+import { Box, Button, Container, Link } from '@mui/material'
+import TextField from '@mui/material/TextField'
+import { putReportApi } from '../../services/report'
+import ReportList from '../../components/ReportList/ReportList'
+
+const Report_s1 = () => {
+  const [Rego, setRego] = useState('')
+  const [VIN, setVIN] = useState('')
+  const [Odometer, setOdometer] = useState('')
+  const [FirstName, setFirstName] = useState('')
+  const [LastName, setLastName] = useState('')
+  const handle_My_Report = async () => {
+    //检察数据
+    const report = {
+      Rego,
+      VIN,
+      Odometer,
+      FirstName,
+      LastName,
+    }
+    const res = await putReportApi(report)
+    console.log(res.data)
+  }
+
+  return (
+    <div color="rgb(235,241,247)">
+      <ReportList />
+      <Container
+        fixed
+        sx={{
+          backgroundColor: 'rgb(235,241,247)',
+          position: 'absolute',
+          left: 304,
+          height: 832,
+          width: 976,
+        }}
+      >
+        <h1
+          style={{
+            top: 40,
+            left: 80,
+            fontFamily: 'Inter',
+            fontStyle: 'normal',
+            fontWeight: 700,
+            fontSize: 26,
+            position: 'absolute',
+          }}
+        >
+          Create New Report
+        </h1>
+
+        <Link
+          href="/report_s1"
+          underline="hover"
+          sx={{ position: 'absolute', left: 80, top: 80, fontSize: 12, fontWeight: 400 }}
+        >
+          {'My Report'}
+        </Link>
+        <p style={{ left: 145, top: 77, position: 'absolute', color: 'rgba(3, 17, 27, 0.4)' }}>
+          &gt;&gt;
+        </p>
+        <h1 style={{ position: 'absolute', fontSize: 12, left: 170, top: 79 }}>
+          Create New Report
+        </h1>
```

Comment:
```suggestion
        <h1 className=''>
          Create New Report
        </h1>
```

To adhere to the principles of readability and maintainability, Don't use inline style css, we use tailwind css and you can check doc on their websites.